specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,165 @@
+
+| Key  | Value | Description  |
+|---|---|---|
+| `event.domain` | browser | Fixed value |
+| `event.name` | | Described in each section below|
+| `event.data` | | A map of key/value pairs, with the keys for each event described in following sections|
+
+
+
+## Page View
+
+<!-- semconv browser.page_view -->
+The event name MUST be `page_view`.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `referrer` | string | Referrer URL for the browser page (`document.referrer`) | `https://en.wikipedia.org/wiki/Main_Page` | Recommended |
+| `type` | int | Browser page type | `0` | Required |
+| `user_consent` | boolean | Whether or not user provided consent on the page (consent for what?) |  | Recommended |
+| [`http.url`](../../../trace/semantic_conventions/http.md) | string | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`. Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless. [1] | `https://en.wikipedia.org/wiki/Main_Page`; `https://en.wikipedia.org/wiki/Main_Page#foo` | Required |
+
+**[1]:** The URL fragment may be included for virtual pages
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page |
+| `1` | virtual_page |
+<!-- endsemconv -->
+
+## Ajax
+
+<!-- semconv browser.ajax -->
+The event name MUST be `ajax`.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `page.url` | string | The URL of the page that made the Ajax call | `https://en.wikipedia.org/wiki/Main_Page` | Recommended |
+| `type` | int | Ajax request type | `0` | Required |
+| [`http.method`](../../../trace/semantic_conventions/http.md) | string | HTTP request method. | `GET`; `POST`; `HEAD` | Required |
+| [`http.request_content_length`](../../../trace/semantic_conventions/http.md) | int | The size of the request payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size. | `3495` | Recommended |
+| [`http.response_content_length`](../../../trace/semantic_conventions/http.md) | int | The size of the response payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size. | `3495` | Recommended |
+| [`http.status_code`](../../../trace/semantic_conventions/http.md) | int | [HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6). | `200` | Conditionally Required: If and only if one was received/sent. |
```

Comment:
I dont understand the conditionally required part.  Isn't this simply optional?

Reply:
It would clarify the behavior in case of timeout or network error (which would have status code 0), also CORS requests that fail CORS rules (eg. a.example.com -> b.example.com without response having correct CORS headers) can have status code 0